packages/documentation-site/patternfly-docs/content/design-guidelines/components/alert/alert.md

```diff
@@ -89,11 +126,29 @@ Use bordered inline alerts to return feedback as the result of a user action wit
  
   <img src="./img/Inline-alert-specific-context.png" alt="inline alert inline with content" />
 
-* **If the alert is being used to indicate errors,** use an error validation summary at the top of the page to indicate what errors are present. The summary should be a quick, general overview. Show an error alert next to the field that is causing the error. The error message should be concise and actionable. It should tell the user what happened and the next steps needed to fit it. 
+* **If the alert is being used to indicate errors,** use an error validation summary at the top of the page to indicate what errors are present. The summary should be a quick, general overview. Show an error alert next to the field that is causing the error. The error message should be concise and actionable. It should tell the user what happened and the next steps needed to fix it. 
+
+  <img src="./img/inline-alert-page-level-with-link-one.png" alt="inline alert inline with link demo 1" /> 
+
+  <img src="./img/inline-alert-page-level-with-link-two.png" alt="inline alert inline with link demo 2" />
+
+* **If multiple errors are present in the same location,** order them from most to least severe. 
+
+  <img src="./img/Multiplealert.png" alt="multiple alerts on a page" />
+
+* **If the alert is being used to indicate errors on a modal,** use an error validation summary at the top of the modal below the title to indicate what errors are present. Show an error alert next to the field that is causing the error. The error message should be concise and actionable. It should tell the user what happened and the next steps needed to fix it. 
+
+  <img src="./img/inline-alert-modal-with-link-one.png" alt="multiple alerts on a modal demo 1" /> 
 
-* If multiple errors are present in the same location, order them from most to least severe. 
+  <img src="./img/inline-alert-modal-with-link-two.png" alt="multiple alerts on a page demo 2" />
 
-  <img src="./img/Multiplealert.png" alt="multiple alerts on a page"/>
+* **If the alert is being used to indicate errors on a modal with an existing alert,** display the new alert above the existing alert.
+
+  <img src="./img/inline-alert-modal-with-existing-alert.png" alt="multiple alerts on a modal with existing alert" />
+
+* **Alert is not needed for a confirmation modal.**
```

Comment:
I think having this bullet here is confusing. I would delete it + the image with it.